examples/README.md

```diff
@@ -0,0 +1 @@
+Please note that the `cfssl` manifests are no longer supported. We have moved on to using `cert-manager` for CockroachDB certificates.
```

Comment:
Lets change this to saying that CFSSL is "deprecated" and that cert-manager is "recommended". :+1: